main/zoe/api/Zoe-Intro.md

```diff
@@ -0,0 +1,109 @@
+# Introduction to Zoe
+
+
+**Note:** Before reading this, you should be familiar with ERTP basics. (**tyg todo**: Add link)
+
+
+This is an introduction to Zoe, Agoric's smart contract framework. The Zoe Guide (**tyg todo**: Add link) has more complete and detailed information, including concepts not covered here. In particular, here we simplify Zoe examples such that there are always two parties in a basic trading/swap contract. We do not discuss more complicated contracts with many participants, such as auctions, or simpler contracts, such as our AutomaticRefund one party demo contract.
```

Comment:
I'm not sure what to do about this, but the whole explanation in terms of "parties" (or participants) leads to a very identity-oriented understanding of what's going on. Instead, everything attributed to "each party" here is really about each offer. Any one participant can make any number of offers. It may be that referring to parties or participants is a good expository simplification. Or it might amplify the identity-oriented misunderstandings that users of other systems will already be drawn to. Where in our expository sequence to we try to cure them of an identity-oriented pov on participating?

Reply:
Hmm. I think it's a bit of a combination of I have a more identity-oriented take than would seem to be correct and that I do think things need to be simplified for the intro; when going over a draft with Kate, she mentioned things like auctions with multiple parties, offers, non-winners getting their offer stakes back even when things go right and the contract completes, etc.  But that's getting away from the goal, which is to give them a basic understanding of what Zoe is/does and our contract execution workflow before they start looking at contracts, which'll start with the most basic ones. 

Curing of identity-orientation would happen in the Zoe Guide, which needs to be redone for many reasons, and/or in comments/exposition on our demo contracts that get away from two parties making an offer. And I need to get myself more in an offer mindset, so I'll try to do at least some rewriting on this to make it moreso.